### PR TITLE
[ODS-4801] MetaEd integration build for Transcript and Transportation Extensions 

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: Utilities/CodeGeneration/EdFi.Ods.CodeGen.Tests/EdFi.Ods.CodeGen.Tests.csproj [hunk 1]
```diff
@@ -44,16 +44,19 @@
...
-    <None Remove="Approval Tests\*.approved.sql" />
+    <None Remove="Approval\*.approved.cs" />
+    <None Remove="Approval\*.approved.xml" />
+    <None Remove="Approval\*.approved.sql" />
   </ItemGroup>
 
   <ItemGroup>
+    <None Update="Approval\ApprovalTests.Generated_File_List.approved.txt">
+      <DependentUpon>ApprovalTests.cs</DependentUpon>
+    </None>
```
Comment (jamessmckay) [inline Utilities/CodeGeneration/EdFi.Ods.CodeGen.Tests/EdFi.Ods.CodeGen.Tests.csproj:59]: why is this needed?

**Review by jamessmckay — CHANGES_REQUESTED** [review]

File: Utilities/CodeGeneration/EdFi.Ods.CodeGen.Tests/EdFi.Ods.CodeGen.Tests.csproj (hunk 1 above)
Comment (semalaiappan) [reply Utilities/CodeGeneration/EdFi.Ods.CodeGen.Tests/EdFi.Ods.CodeGen.Tests.csproj]: Thanks  .  I removed it

**Review by jamessmckay — APPROVED** [review]
